i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md

```diff
@@ -3,6 +3,8 @@ id: centreon-commercial-extensions
 title: Extensions Commerciales
 ---
 
+import expAll from './src/expand_collapse/expand_collapse.js';
```

Comment:
Rather than importing this in the md file, because md can not process js files.

Follow this [documentation](https://docusaurus.io/docs/next/api/docusaurus-config#scripts)